include/roboteam_ai/interface/widgets/PlaysWidget.hpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by john on 4/30/20.
+//
+
+#ifndef RTT_PLAYSWIDGET_HPP
+#define RTT_PLAYSWIDGET_HPP
+
+#include <QtWidgets/QVBoxLayout>
+#include <QtWidgets/QTextEdit>
+
+namespace rtt::ai::interface {
+
+    class PlaysWidget : public QTextEdit {
```

Comment:
Where is the class documentation?

Reply:
it's a temporary class i didn't put too much effort into it.

Reply:
Plus it's pretty self-explanatory